docs/usage/php.md

```diff
@@ -18,3 +18,35 @@ Renovate supports upgrading dependencies in PHP's `composer.json` files and thei
 ## Enabling
 
 Either install the [Renovate App](https://github.com/apps/renovate) on GitHub, or check out [Renovate OSS](https://github.com/renovatebot/renovate) for self-hosted.
+
+## Private packages
+
+If you are using a [privately hosted Composer package](https://getcomposer.org/doc/articles/authentication-for-private-packages.md) you can pass the credentials via the [`hostRules`](https://docs.renovatebot.com/configuration-options/#hostrules) configuration.
+
+```json
+{
+    "hostRules": [{
+        "hostName": "some.vendor.com",
+        "hostType": "packagist",
+        "username": "<your-username>",
+        "password": "<your-password>"
+    }]
+}
+```
+
+If you don't wish for all users of the repository to be able to see the unencrypted token, you can encrypt it with Renovate's public key instead, so that only Renovate can decrypt it.
```

Comment:
```suggestion
This host rule is best added to the bot's `config.js` config so that it is not visible to users of the repository. If you are using the hosted WhiteSource Renovate App then you can encrypt it with Renovate's public key instead, so that only Renovate can decrypt it.
```